web/packages/teleterm/package.json

```diff
@@ -42,9 +42,9 @@
     "@types/node-forge": "^1.3.11",
     "@types/tar-fs": "^2.0.4",
     "@types/whatwg-url": "^11.0.5",
-    "electron": "31.0.1",
+    "electron": "31.1.0",
```

Comment:
I'll have to verify this update tomorrow.

Reply:
Installed 17.0.0-dev.ravicious.3 on all four supported systems, everything seems to work fine. Patch notes don't indicate any changes that are relevant for us.

https://releases.electronjs.org/release/compare/v31.0.1/v31.1.0